developers/sp1-blobstream-deploy.md

```diff
@@ -0,0 +1,35 @@
+---
+next:
+    text: "Celestia-node key"
+    link: "/developers/celestia-node-key"
+---
+
+# New SP1 Blobstream deployments
+
+This document provides instructions for deploying SP1 Blobstream to a new chain.
+
+## Deploying the contracts
+
+To deploy SP1 Blobstream to a new chain, follow these steps:
+
+1. Clone the sp1-blobstream repository:
+
+```shell
+git clone https://github.com/succinctlabs/sp1-blobstream
+cd sp1-blobstream
+```
+
+2. Follow the deployment instructions in the
+[sp1-blobstream README](https://github.com/succinctlabs/sp1-blobstream?tab=readme-ov-file#deployment).
+
+3. If you're deploying on a chain where there isn't a canonical verifier listed in the
+[SP1 contract addresses](https://github.com/succinctlabs/sp1/blob/main/book/onchain-verification/contract-addresses.md), you'll need to:
+
+   a. Deploy your own SP1 Verifier from `sp1-contracts` matching your `sp1-sdk` version.
```

Comment:
@ratankaliani any instructions on how to do this? like a link to the SP1 verifier, how to build it, where to download it.

Also, would it also need to whitelist a custom prover for example, or similar?

Reply:
The contracts for each SP1 version can be found here: https://github.com/succinctlabs/sp1-contracts

Don't think we need to include instructions on how to build it.

Any whitelisting for custom provers would happen in the application's smart contracts. (e.g. Having an `approvedProvers` mapping)

Reply:
addressed in https://github.com/celestiaorg/docs/pull/1680/commits/5856e10eefc785f25af549b57d2e93c7e1179868